RocketChat/Rocket.Chat.ReactNative
# regression(Android): Poor performance in messages list

## Proposed changes

In this PR #5251 we removed RefreshControl, however it caused a loss of performance when using the app.
This PR removes RefreshControl without impacting performance.

## Issue(s)	


## How to test or reproduce

Navigate to a room and check whether the performance when navigating between messages is acceptable.

## Screenshots

## Types of changes




## Checklist



## Further comments

Jira: [NATIVE-410]


[NATIVE-410]: https://rocketchat.atlassian.net/browse/NATIVE-410?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
